Castanet/Feed.php

```diff
@@ -166,6 +206,41 @@ public function setItunesBlock($itunes_block)
 	}
 
 	// }}}
+	// {{{ public function setAtomLink()
+
+	public function setAtomLink($atom_link)
+	{
+		$this->atom_link = strval($atom_link);
+	}
+
+	// }}}
+	// {{{ public function setItunesCategories()
+
+	public function setItunesCategories($itunes_categories)
```

Comment:
This change allows multiple top-level categories but it's probably more important to allow multiple optional sub-categories as iTunes ignores all top-level categories after the first one.

With this change, it's not possible to add a subcategory.